mlflow/tracking/fluent.py

```diff
@@ -152,6 +152,14 @@ def create_experiment(name, artifact_location=None):
     return get_service().create_experiment(name, artifact_location)
 
 
+def delete_experiment(experiment_id):
+    return get_service().delete_experiment(experiment_id)
```

Comment:
We should have a comment here as well. We could also consider not exposing delete & restore in the fluent API (as they are a bit rarer to use).